src/locales/en.json

```diff
@@ -4,60 +4,60 @@
     "liberty.ls.thumbs.up": "Liberty LS $(thumbsup)",
     "liberty.ls.thumbs.down": "Liberty LS $(thumbsdown)",
     "liberty.ls.starting": "Language Server for Liberty is starting...",
-    "liberty.ls.started": "Language Server for Liberty started",
-    "liberty.ls.failstart": "Language Server for Liberty failed to start",
+    "liberty.ls.started": "Language Server for Liberty started.",
+    "liberty.ls.failstart": "Language Server for Liberty failed to start.",
     "jakarta.ls": "LSP4Jakarta",
     "jakarta.ls.output.dropdown": "Language Support for Jakarta EE",
     "jakarta.ls.thumbs.up": "LSP4Jakarta $(thumbsup)",
-    "jakarta.ls.started": "Language Server for Jakarta started",
+    "jakarta.ls.started": "Language Server for Jakarta started.",
     "no.liberty.projects.found": "No liberty projects found.",
-    "starting.liberty.dev.on": "Starting liberty dev on {0}",
-    "cannot.start.liberty.dev": "Cannot start liberty dev on an undefined project",
-    "stopping.liverty.dev.on": "Stopping liberty dev on {0}",
-    "liberty.dev.not.started.on": "liberty dev has not been started on {0}",
-    "cannot.stop.liberty.dev.on.undefined": "Cannot stop liberty dev on an undefined project",
+    "starting.liberty.dev.on": "Starting liberty dev on {0}.",
+    "cannot.start.liberty.dev": "Cannot start liberty dev on an undefined project.",
+    "stopping.liverty.dev.on": "Stopping liberty dev on {0}.",
+    "liberty.dev.not.started.on": "liberty dev did not start on {0}.",
```

Comment:
@dmuelle I am wondering if we should be saying "Liberty dev mode" instead of "liberty dev" in each of these messages. What are your thoughts?

Reply:
i agree

Reply:
updated